servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilderTest.java

```diff
@@ -85,8 +85,9 @@ private static void hostToCharSequenceFunction(String hostNamePrefix, String hos
                         .build())
                 .listenBlockingAndAwait((ctx, request, responseFactory) -> responseFactory.ok());
              BlockingHttpClient client =
-                     forResolvedAddress(hostNamePrefix + hostName + hostNameSuffix + (port == null ? "" : port),
-                             u -> serverCtx.listenAddress())
+                     forSingleAddress(
+                             GlobalDnsServiceDiscoverer.mappingServiceDiscoverer(u -> serverCtx.listenAddress()),
```

Comment:
Cute! I wondered what you were adding that for, it seemed unrelated.

Reply:
The goal was to remove all static factories from `DefaultSingleAddressHttpClientBuilder` to keep the only one ctor. To achieve that, I had to move SD for resolved addresses out or `DefaultSingleAddressHttpClientBuilder`. It required change in this test because there is no more `DefaultSingleAddressHttpClientBuilder.forResolvedAddress`

Reply:
Actually, because this is `DefaultSingleAddressHttpClientBuilderTest`, let me just use `DefaultSingleAddressHttpClientBuilder` ctor here.